src/components/DataTable/DataTable.tsx

```diff
@@ -30,12 +30,14 @@ import {
   ViewDialog,
   ContactRejectDialogProps,
   RevokeKeyDialogProps,
+  TransactionEditDialogProps,
+  DestinationEditDialogProps,
 } from '@/components';
 
 interface DataTableProps<TData, TValue> {
   columns: ColumnDef<TData, TValue>[];
   data: TData[];
-  EditDialog?: React.ComponentType<ContactEditDialogProps>;
+  EditDialog?: React.ComponentType<ContactEditDialogProps | TransactionEditDialogProps | DestinationEditDialogProps>;
```

Comment:
Hmm.. do we have any naming convention? Im not familiar with React as much as Angular. But it looks like two different naming conventions. Sometimes field starts with lower case and sometimes with uppercase. 

Shouldn't be `columns, data` and `editDialog` or `Columns, Data, EditDialog`?

Reply:
As far as I know, the `EditDialog` starts with uppercase letter because it is a React component - and React components supposed to be named like that. 
As this particular line of code - looks good to me. (And I'm familiar with React 🙈)

Reply:
React have a naming convention. Components should starts with uppercase